### PR TITLE
Fix prometheus link

### LINKED ISSUE #3110: Incorrect prometheus and owner on store page
The promoted Prometheus on the /store page is the incorrect one. It should be:

https://jujucharms.com/prometheus/1

The GUI also shows this as owned by "Charmers" when it should be showing "prometheus-charmers"

https://api.jujucharms.com/charmstore/v5/prometheus/meta/owner

### PR DESCRIPTION
**Pull request by makyo**
Fixes #3110

### CODE REVIEW

File: jujugui/static/gui/src/app/components/store/store.js [hunk 1]
```diff
@@ -507,12 +507,12 @@ class Store extends React.Component {
           </li>
           <li className="three-col last-col featured-entity">
             <span className="featured-entity__link link"
-              data-entity="u/canonical-bootstack/prometheus"
+              data-entity="prometheus/1"
```
Comment (hatched) [inline jujugui/static/gui/src/app/components/store/store.js]: maybe no /1 so it'll pick up the latest release?

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/7936/

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/7937/

**Comment by makyo** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**